### ISSUE TITLE
ACC-264.1 (Level A) _ Modify Modules _ Button _ Heading level is read for a button

### ISSUE THREAD

**Issue by shubha1090** [kyma-project/busola#3141]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	Medium
Affected UI	Button
Affected Check Point:	ACC-264.1
Short text:	ACC-264.1 (Level A) _ Modify Modules _ Button _ Heading level is read for a button

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. Now Select ‘Modify Modules’ button which opens ‘Modules’ page.
•	Open JAWS 2024 screen reader and turn on VPC mode.
•	Navigate to ‘Information’ button within main header region in reading mode using arrow keys.

3. Issue Type: Manual
Observed Behavior:
On accessing the ‘Information’ button within main header region in reading mode using arrow keys, information about heading level i.e. ‘heading level 3 button Unlabeled 0’  is announced by the screen reader whereas it is a button and not a heading and no other heading levels are defined in the page which confuses the user. 
Screenshot:
 

Expected Behavior:
Heading levels must be defined only for headings and not for buttons and heading level announcement for this button must be removed from the speech output. 

Note: 
This issue exists across the application and thus fix should be provided for all the occurrences.
________________________________________________________________________
Kindly refer to attached error and issue processing document for more details.
Regards,
Shubha

**Comment by shubha1090** [comment]
[ACC-264.1 (Level A) _ Modify Modules _ Button _ Heading level is read for a button.docx](https://github.com/user-attachments/files/16659931/ACC-264.1.Level.A._.Modify.Modules._.Button._.Heading.level.is.read.for.a.button.docx)

[Issue processing-09.2021 (002).docx](https://github.com/user-attachments/files/16659933/Issue.processing-09.2021.002.docx)
